common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -38,4 +42,15 @@ object CollectionsManager {
 
   // We could use `ValidationNel`s here, but that's overkill
   def isValidPathBit(s: String) = if (s.contains(delimiter)) false else true
+
+  // TODO: Find something that works off a palette, this is terrible.
+  def stringToCssRgb(s: String): String = {
+    val hash = s.hashCode
+    val r = (hash & 0xFF0000) >> 16
+    val g = (hash & 0x00FF00) >> 8
+    val b = hash & 0x0000FF
+    s"rgb($r, $g, $b)"
```

Comment:
Hardcoding colours would probably better so we can pick colours that work well together and avoid the "angry fruit salad" syndrom. Just talked to @tsop14 about it and she's going to be looking into it.